lib/jsdom/living/custom-elements/ElementInternals-impl.js

```diff
@@ -15,6 +19,30 @@ class ElementInternalsImpl {
     return shadow;
   }
 
+  get labels() {
+    if (!this._targetElement._ceDefinition.formAssociated) {
+      throw DOMException.create(this._targetElement._globalObject, [
+        "ElementInternals.labels is only supported for form-associated custom elements",
```

Comment:
Nit: don't use `InterfaceName.property` for non-static properties. Prefer something like `ElementInternal's labels property`.